src/Qir/Runtime/lib/Simulators/FullstateSimulator.cpp

```diff
@@ -99,7 +100,11 @@ namespace Quantum
         }
 
         const QUANTUM_SIMULATOR handle = 0;
-        unsigned simulatorId = -1;
+
+        using TSimulatorId = unsigned;      // TODO: Use `void*` or a fixed-size integer, starting in native simulator (breaking change).

```

Comment:
Do you need to take care of this ToDo before merging?

Reply:
Resolved in a different discussion earlier.